api/v1beta2/integrationtestscenario_types.go

```diff
@@ -103,6 +105,9 @@ type ResolverRef struct {
 	// the chosen resolver.
 	// +required
 	Params []ResolverParameter `json:"params"`
+	// ResolverType defines the type of resolver. It can either be "pipeline" or "pipelinerun"
+	// but defaults to "pipeline" if no value is set
```

Comment:
Since it seems like this field was added to required fields when generating the CRD (on this [line](https://github.com/konflux-ci/integration-service/pull/1091/files#diff-e6ffed760c39b7a2330d0fa640f5d8e96d6acafcfaf3c80a907787f353d2c70cR561)), let's try adding the 
```
// +optional
```
Below this line like it's done e.g. [here](https://github.com/konflux-ci/application-api/blob/5a9670bf76794f54fdd21009a2a0920c0db0811f/api/v1alpha1/component_types.go#L99).

Reply:
This should resolve the e2e testing failures.

Reply:
Thanks so much for noticing it.